react/testing.md

```diff
@@ -71,27 +71,33 @@ The `data-spec` attribute is specific to testing and not tied to presentation li
 </div>
 ```
 
-Tests using `getSingleSpecWrapper` would look like:
+Tests using `getSpecWrapper` would look like:
```

Comment:
so the output of getSpecWrapper is different depending on what we get back? single node or array of nodes?